.github/workflows/update.yml

```diff
@@ -8,17 +8,26 @@ jobs:
     name: Deploy docs
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout main
+      - name: Checkout
         uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Checkout LFS objects
+        run: git lfs pull
 
       - uses: actions/setup-python@v2
         with:
           python-version: 'pypy-2.7'
 
-      - name: Build pages and publish to S3
-        uses: dmitry-spodarets/mkdocs-s3-publish-action@main
+      - name: Build pages
+        uses: Exabyte-io/mkdocs-s3-publish-action@main
+
+      - name: S3 Sync
+        uses: unfor19/install-aws-cli-action@master
+      - run: aws s3 sync site/ s3://docs.mt3ra.com/
+        shell: bash
         env:
-          S3_LOCATION: ${{ secrets.S3_LOCATION }}
-          S3_ACCESS_KEY: ${{ secrets.S3_ACCESS_KEY }}
-          S3_SECRET_KEY: ${{ secrets.S3_SECRET_KEY }}
-          S3_BUCKET: ${{ secrets.S3_BUCKET }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.S3_ACCESS_KEY }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.S3_SECRET_KEY }}
+          AWS_DEFAULT_REGION: ${{ secrets.S3_LOCATION }}
```

Comment:
<img src="https://media2.giphy.com/media/jnKNfpausKbgQz2bEV/giphy.gif"/>

Reply:
Let's add a note about these to https://exabyte.atlassian.net/wiki/spaces/~660192581/pages/2419982345/Public+Documentations pls.

Same as https://github.com/Exabyte-io/swaggerui/pull/2/files#r774852330.